### PR TITLE
Google Safe Browsing v2 bug fix

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/37108

## Description
Fixed an issue where the url command returned an error where no results were found.

#### The Error:
image

#### Fix:
image

## Minimum version of Cortex XSOAR
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py [hunk 1]
```diff
@@ -150,7 +150,10 @@ def url_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
 
     result = client.url_request(client.client_body, url)
 
-    if not result or result.get('StatusCode'):
+    if not result:
+        return f'No information was found for url {url}'
```
Comment (Itay4) [inline Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py:154]: need to return dbot score 0 in this case

**Review by Itay4 — COMMENTED** [review]
please add test playbook which checks for this kind of url

**Comment by Shellyber** [comment]
@roysagi Please review the RN

File: Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2_test.py [hunk 2]
```diff
@@ -145,3 +145,18 @@ def test_command_url(mocker):
     assert output.get(dbot_key, [])[0].get('Score') == 3
     assert output.get(dbot_key, [])[0].get('Reliability') == DBotScoreReliability.B
     assert url_command[0].to_context().get('Contents') == URL_CONTENTS
+
+
+def test_url_not_found(mocker):
```
Comment (Itay4) [inline Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2_test.py:150]: please add docstring

File: Packs/GoogleSafeBrowsing/ReleaseNotes/2_0_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Google Safe Browsing v2
+- Fixed an issue where the url command returned an error where no results were found.
+- Updated docker image to the latest version.
```
Comment (Itay4) [inline Packs/GoogleSafeBrowsing/ReleaseNotes/2_0_1.md:5]: please mention the docker image we updated to

**Review by Itay4 — APPROVED** [review]

**Pull request merged**